qa/integration/specs/cli/prepare_offline_pack_spec.rb

```diff
@@ -87,6 +87,7 @@
                                 .stderr_and_stdout.split("\n")
                                 .delete_if do |line|
                                   line =~ /cext|├──|└──|logstash-integration|JAVA_OPT|fatal|^WARNING|^warning: ignoring JAVA_TOOL_OPTIONS|^OpenJDK 64-Bit Server VM warning|Option \w+ was deprecated|Using LS_JAVA_HOME defined java|Using JAVA_HOME defined java|Using system java: |\[\[: not found/ ||
+                                  line =~ /^Warning: no jvm.options file found|^Processing jvm.options file at/
```

Comment:
Reviewer note: this is the only line that didn't merge cleanly. The upstream PR appended these matcher clauses to the line above, which had a different form to accommodate JAVA_HOME etc.